scripts/secnetperf.ps1

```diff
@@ -225,19 +232,29 @@ if ($isWindows) {
 Configure-DumpCollection $Session
 
 # Install any dependent drivers.
-if ($useXDP) { Install-XDP $Session $RemoteDir }
+if ($useXDP -and $isWindows) { Install-XDP $Session $RemoteDir }
 if ($io -eq "wsk") { Install-Kernel $Session $RemoteDir $SecNetPerfDir }
 
 if (!$isWindows) {
     # Make sure the secnetperf binary is executable.
     Write-Host "Updating secnetperf permissions"
+    $GRO = "on"
+    $ENABLE_XDP=0
+    if ($io -eq "xdp") {
+        $GRO = "off"
+        $ENABLE_XDP=1
+    }
     Invoke-Command -Session $Session -ScriptBlock {
         $env:LD_LIBRARY_PATH = "${env:LD_LIBRARY_PATH}:$Using:RemoteDir/$Using:SecNetPerfDir"
         chmod +x "$Using:RemoteDir/$Using:SecNetPerfPath"
+        sudo sh -c "ethtool -K eth0 generic-receive-offload $Using:GRO"
+        $env:MSQUIC_ENABLE_XDP=$Using:ENABLE_XDP
     }
     $fullPath = Repo-Path $SecNetPerfDir
     $env:LD_LIBRARY_PATH = "${env:LD_LIBRARY_PATH}:$fullPath"
     chmod +x "./$SecNetPerfPath"
+    sudo sh -c "ethtool -K eth0 generic-receive-offload $GRO"
+    $env:MSQUIC_ENABLE_XDP=$ENABLE_XDP
```

Comment:
Reset this in cleanup?

Reply:
cleanup phase (beginning of secnetperf.ps1) cannot know the default state of GRO.
So  I specify the feature off/on for xdp/others.

Reply:
Is there no way to query the state at the beginning and save it? But really, as I said in the comment above, we should figure out a programmatic solution for just the XDP socket, instead of a system-wide setting.

Reply:
Possible to just query, but need external file to save the state for the case if the workload is killed during the run.